ansible/TestbedProcessing.py

```diff
@@ -457,6 +542,10 @@ def makeVeos(data, veos, devices, outfile):
                     try:
                         ansible_host = devices.get(host.lower()).get("ansible").get("ansible_host")
                         entry += "\tansible_host=" + ansible_host.split("/")[0]
+                        if dev.get("device_type") == "DevSonic":
+                            entry += "\ttype=" + dev.get("type")
+                            entry += "\thwsku=" + dev.get("hwsku")
+                            entry += "\tcardtype=" + dev.get("cardtype")
```

Comment:
We use 'card_type' as the host variable to distinguish between linecard and supervisor card. Can we change 'cardtype' to 'card_type'.